src/presentation/webapp/sp-emergency-responses/EmergencyResponsesSyncHomePage.tsx

```diff
@@ -27,12 +26,46 @@ export const EmergencyResponsesSyncHomePage: React.FC<EmergencyResponsesSyncHome
     const rules = useSyncRulesList(emergencyType);
     const [isRunning, runSyncRule] = useSyncRulesExecuter({ emergencyType, logs });
 
+    const executeRules = React.useCallback(
+        (rules: SynchronizationRule[]) =>
+            _.orderBy(rules, rule => rule.code === "EFH_METADATA_ORGUNITS", "desc").forEach(rule => {
+                console.debug("Running: " + rule.name);
+                runSyncRule(rule);
```

Comment:
runSyncRule is an async function. Being called in a forEach, that will call them in parallel. Probably not what we want. Adding await it's not sufficient, we'll need to convert the forEach to a for-loop, or probably better, use `promiseMap`.